CharlieBackend.Business/Services/AccountService.cs

```diff
@@ -153,6 +153,24 @@ public async Task<bool> DisableAccountAsync(long id)
             }
         }
 
+        public async Task<bool> EnableAccountAsync(long id)
+        {
+            try
+            {
+                var isSucceeded = await _unitOfWork.AccountRepository.EnableAccountAsync(id);
+
+                await _unitOfWork.CommitAsync();
+
+                return isSucceeded;
+            }
+            catch
+            {
+                _unitOfWork.Rollback();
+
+                return false;
+            }
```

Comment:
Do we really need to enclose this logic inside a `try catch` block?

Reply:
I've removed it. Can't figure it out, why it has been added.